index.js

```diff
@@ -51,87 +50,86 @@ function RFID (hardware, options, callback) {
   self.hardware = hardware;
   self.ready = false;
   self.irq = hardware.digital[2];
-  self.irqcallback = function () {
-    self.emit('irq', null, 0);
-  };
-  self.irq.on('fall', self.irqcallback);
```

Comment:
This was just convoluted. Instead, I opted to use `self.irq.once('low`, function ready() {...})` any place that we were waiting for an interrupt.